### PR TITLE
Update SMART extension documentation

### PR DESCRIPTION
**Pull request by raphaelyancey**
The documentation in the comments of the script (https://github.com/librenms/librenms-agent/raw/master/snmp/smart) is clearer regarding the device line format.

Also reformatted a bit and fixed the sudoers allowed commands.

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: doc/Extensions/Applications.md [hunk 1]
```diff
@@ -1629,52 +1629,56 @@ hide_monitoring_account = With this Boolean you can hide the Account which you
...
 ```
 #This is a comment
 cache=/var/cache/smart
-smartctl=/usr/bin/env smartctl
-useSN=1
+smartctl=/usr/local/sbin/smartctl
+useSN=0
 ada0
-ada1
+da5 /dev/da5 -d sat
```
Comment (SourceDoctor) [inline doc/Extensions/Applications.md]: not really agree with your changes.
please let Options on default values in example
Also keep `ada1` as second example for direct device access.

**Comment by SourceDoctor** [comment]
i personally would keep `/usr/bin/env smartctl` as it is,

if smartctl is located on /usr/sbin/ , /usr/local/sbin/ depends on system.
So a change here seems to be not useful, because the next one will change it matching to his system, then.

File: doc/Extensions/Applications.md (hunk 1 above)
Comment (raphaelyancey) [reply doc/Extensions/Applications.md]: What helped me the most was actually:
```
A disk line is can be as simple as just a disk name under /dev/. Such as in the config above
The line "ada0" would resolve to "/dev/ada0" and would be called with no special argument. If
a line has a space in it, everything before the space is treated as the disk name and is what
used for reporting and everything after that is used as the argument to be passed to smartctl.
```

... which describe the format very clearly. Example lines aren't clear on their own, pretty opaque to say the least.

Anyway this is copypasted from the script itself, don't you think it should match?

**Comment by raphaelyancey** [comment]
> i personally would keep `/usr/bin/env smartctl` as it is,
> 
> if smartctl is located on /usr/sbin/ , /usr/local/sbin/ depends on system.
> So a change here seems to be not useful, because the next one will change it matching to his system, then.

I agree and will change it.

File: doc/Extensions/Applications.md (hunk 1 above)
Comment (SourceDoctor) [reply doc/Extensions/Applications.md]: if you update Documentation, it's very welcome.
Examples should match default values.
It's correct the example in agent script should be fixed also.

Netherless the second example device 'ada1' should stay

Comment (raphaelyancey) [reply doc/Extensions/Applications.md]: Okay, will do.

> Netherless the second example device 'ada1' should stay

What information does `ada1` add exactly? As `ada0` is already a "direct device access" example in the same format.

Comment (SourceDoctor) [reply doc/Extensions/Applications.md]: ada1 is an second disc, for seeing how the next disc has to be added

**Comment by raphaelyancey** [comment]
Looks like the list numbering is messed up when rendered in Markdown :/

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-62-release-changelog-march-2020/11423/1

**Pull request merged**